### ISSUE TITLE
pip install with only-binary fails

### ISSUE THREAD

**Issue by owillebo** [DiffSK/configobj#211]
Running Python 3.9.1 on Windows 10.

Installing configobj fails when pip --only-binary option is used.

Is there any binary stuff in the configobj package?

If not, please make configobj install with --only-binary option.

```
C:\projects\DNW\Code\prerequisites_3>python --version
Python 3.9.1

C:\projects\DNW\Code\prerequisites_3>python -m pip install --only-binary=:all: configobj
ERROR: Could not find a version that satisfies the requirement configobj
ERROR: No matching distribution found for configobj

C:\projects\DNW\Code\prerequisites_3>python -m pip install configobj
Collecting configobj
  Using cached configobj-5.0.6-py3-none-any.whl
Requirement already satisfied: six in c:\program files (x86)\python39-32\lib\site-packages (from configobj) (1.15.0)
Installing collected packages: configobj
Successfully installed configobj-5.0.6
```

**Comment by robdennis** [comment]
My very naive guess is that this error is due to the only uploaded artifacts in pypi are source tarballs.

If this can be addressed via generating a wheel and uploading it, I'll give that a try in the new dev cycle.

Otherwise I may need someone who actually understands how pip works now to unblock this

**Comment by robdennis** [comment]
5.0.7 does have a wheel, please re-open / make a new ticket if this doesn't address your needs.
